webpack.config.js

```diff
@@ -46,8 +47,15 @@ module.exports = [
                 '@babel/preset-env',
                 {
                   modules: false,
+                  useBuiltIns: 'entry',
                   targets: {
-                    browsers: ['last 2 versions'],
+                    browsers: [
+                      'Chrome >= 62',
+                      'Edge >= 15',
+                      'FireFox >= 56',
```

Comment:
According to https://babeljs.io/docs/plugins/preset-env/#targets I would have expected lowercase browsernames

Reply:
Makes sense. I was toying around with browser list originally.